### PR TITLE
Add note to the comment

### PR DESCRIPTION
**Pull request by MatkovIvan**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Related PRs or issues

https://github.com/MicrosoftDocs/appcenter-docs/issues/463

### CODE REVIEW

File: AppCenter/AppCenter/MSAppCenter.h [hunk 1]
```diff
@@ -83,6 +83,7 @@
 
 /**
  * Enable or disable the SDK as a whole. In addition to AppCenter resources, it will also enable or disable all registered services.
+ * The state is stored on disk, so it won't change in following app launches.
```
Comment (dhei) [inline AppCenter/AppCenter/MSAppCenter.h]: Let me give it a try - "The state is persist on disk across app launches."

Comment (guperrot) [reply AppCenter/AppCenter/MSAppCenter.h]: Let's not use disk as already discussed on android pr and docs pr

**Comment by MatkovIvan** [comment]
@jaelim-ms Fixed in #1307. Changes in comments don't affect tests anyway.

File: AppCenter/AppCenter/MSService.h [hunk 2]
```diff
@@ -6,20 +6,21 @@
 @protocol MSService <NSObject>
 
 /**
- *  Enable/disable this service.
+ * Enable/disable this service.
+ * The state is persisted in the device's storage across application launches.
  *
- *  @param isEnabled whether this service is enabled or not.
+ * @param isEnabled whether this service is enabled or not.
```
Comment (ElektrojungeMS) [inline AppCenter/AppCenter/MSService.h]: As we're touching this comment, how about

* capitalize "Whether"
* capitalize "A" in "A boolean..."

Actually, thinking of it, why not rewrite both comments for `isEnabled` into a full sentence? How about "A BOOL that indicates if this service is enabled or not"?

**Pull request merged**